packages/core-typings/src/IMessage/MessageAttachment/MessageAttachmentBase.ts

```diff
@@ -4,6 +4,7 @@ export type MessageAttachmentBase = {
 	ts?: Date;
 	collapsed?: boolean;
 	description?: string;
+	text?: string;
```

Comment:
I see `text` is already defined at `MessageAttachmentDefault` , so not sure it should be moved to here or we should keep it there..

idk why having both types "Default" and "Base" for "MessageAttachment" to be honest.. 🤔  maybe @ggazzo can give some background on why both exists.